tests/functional/change_password.js

```diff
@@ -6,45 +6,48 @@ define([
   'intern',
   'intern!object',
   'tests/lib/helpers',
-  'tests/functional/lib/helpers'
-], function (intern, registerSuite, TestHelpers, FunctionalHelpers) {
-  var config = intern.config;
-  var SIGNIN_URL = config.fxaContentRoot + 'signin';
-
-  var FIRST_PASSWORD = 'password';
-  var SECOND_PASSWORD = 'new_password';
-  var email;
-
-  var ANIMATION_DELAY_MS = 500;
-
-  var thenify = FunctionalHelpers.thenify;
-
-  var clearBrowserState = FunctionalHelpers.clearBrowserState;
-  var click = FunctionalHelpers.click;
-  var createUser = FunctionalHelpers.createUser;
-  var denormalizeStoredEmail = FunctionalHelpers.denormalizeStoredEmail;
-  var fillOutChangePassword = FunctionalHelpers.fillOutChangePassword;
-  var fillOutSignIn = FunctionalHelpers.fillOutSignIn;
-  var noSuchElementDisplayed = FunctionalHelpers.noSuchElementDisplayed;
-  var openPage = FunctionalHelpers.openPage;
-  var testElementExists = FunctionalHelpers.testElementExists;
-  var testElementTextEquals = FunctionalHelpers.testElementTextEquals;
-  var testSuccessWasShown = FunctionalHelpers.testSuccessWasShown;
-  var type = FunctionalHelpers.type;
-  var visibleByQSA = FunctionalHelpers.visibleByQSA;
-
-  var setupTest = thenify(function (options) {
-    options = options || {};
-    var signUpEmail = options.signUpEmail || email;
-    var signInEmail = options.signInEmail || email;
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/selectors'
+], function (intern, registerSuite, TestHelpers, FunctionalHelpers, selectors) {
+  'use strict';
+
+  const config = intern.config;
+  const SIGNIN_URL = config.fxaContentRoot + 'signin';
+
+  const ANIMATION_DELAY_MS = 500;
+  const FIRST_PASSWORD = 'password';
+  const SECOND_PASSWORD = 'new_password';
+
+  let email;
+
+  const {
```

Comment:
Now that we are on Node 6, we can use destructuring!

Reply:
cool 👍